_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,70 @@
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a great way to sort your subscriptions and a mute button, on top of the usual smaller improvements and fixes<!-- more -->
+
+
+## Folders and fixed Feeds
+
+Since moving images say more than my words, please have a look at them:
+
+<video src="/img/folders.mp4" class="no-flow img-responsive" width="220" height="140" autoplay loop></video>
+
+For those who prefer to read, this is for you: NewPipe now supports grouping subscriptions into folders. All the recent videos of those channels then appear in one re-loadable feed when you press on the folder. While [@mauriciocolli](https://github.com/mauriciocolli) implemented that, he also improved the `What's New` sidebar feed. With improved I mean finally fixing it: The videos are showing in the correct order of the upload, not the first video each channel in a random order. And on top of that, live streams have now a viewer count in those feeds!
+
+An additional feature squashed into this change is the implementation of a so called _fast-mode_. If you enable it in the `Content` menu, those feeds will load much faster from the services which provide a so called _dedicated feed_. The downside of that is that those feeds contain less information, which might result in duration or the live status of videos not shown in the preview. The decision is up to you: Living on the speeding edge and missing information or taking it nice and slow, but knowing **everything**.
+
+
+## Mute button
+
+This release doesn't stop at folders though. Thanks to [@karkaminski](https://github.com/karkaminski), NewPipe now has mute button. [@B0pol](https://github.com/B0pol) formed it further into the shape you can experience it in now. _"But where can I find it"_ you ask. Fear not, for I have prepared a gallery of screenshots. To access the pop-up and background player menus shown, you need to press on the notification from NewPipe.
+
+<img class="no-flow vertical" src="/img/screenshots/mute_main.jpg"/>
+<div class="media-row media-items-2">
+<img class="no-flow img-responsive" src="/img/screenshots/mute_pop.jpg"/>
+<img class="no-flow img-responsive" src="/img/screenshots/mute_back.jpg"/>
+</div>
+
+
+## Smaller Improvements
+
+[@comradekingu](https://github.com/comradekingu) took care of the English base strings in this release and made sure they are consistent. Further improving the user experience was also [@XiangRongLins](https://github.com/XiangRongLin) goal, he made sure that the seek duration jumped visibly to an even one when the fast inexact search option gets activated. Previously the user wouldn't see this directly, but experience when seeking for- or backwards. Thanks to [@opusforlife2](https://github.com/opusforlife2), two settings moved from `Appearance` to `Content`, where they belonged all along: `Show 'Next'` and `Content of main page`.
```

Comment:
[@comradekingu](https://github.com/comradekingu) took care of the English base strings in this release and made sure they are consistent. Further improving the user experience was also [@**XiangRongLin's**](https://github.com/XiangRongLin) goal, **he** _(he or she? I am not sure whether Xiang or Lin is the forename here [AFAIK Xiang can be used by males and females and I've had a classmate and she was called Lin])_ made sure that the seek duration jumped visibly to an even one when the fast inexact search option gets activated. Previously the user wouldn't see this directly, but experience when seeking for- or backwards. Thanks to [@opusforlife2](https://github.com/opusforlife2), two settings moved from `Appearance` to `Content`, where they belonged all along: `Show 'Next'` and `Content of main page`.

Reply:
If you don't know, you can word the sentence differently. Here, who fits good imo: ``Further improving the user experience was also @XiangRongLin's goal, **who** made sure that the seek duration jumped visibly to an even one when the fast inexact search option gets activated.

Reply:
I think @B0pol's suggestions is good

Reply:
Previously the user could get "stuck" when seeking in the main video with 5 second seek duration and inexact seek active. That was the main reason for making the seek duration jump to the next "valid" value when activating inexact seek.

See TeamNewPipe/NewPipe#3127